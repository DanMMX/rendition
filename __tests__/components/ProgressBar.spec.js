import React from 'react';
import renderer from 'react-test-renderer';
import Provider from '../../src/components/Provider'
import ProgressBar from '../../src/components/ProgressBar'

test('ProgressBar renders correctly', () => {
  const value = 50;
  const component = renderer.create(
    <Provider>
      <ProgressBar value={value}>
        {value}%
      </ProgressBar>
    </Provider>,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
