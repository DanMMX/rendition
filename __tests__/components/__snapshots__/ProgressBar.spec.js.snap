// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProgressBar renders correctly 1`] = `
<div
  className="sc-bdVaJa jKopLr"
>
  <div
    className="sc-EHOje brTwkL sc-ifAKCX jITXeW"
    type={undefined}
  >
    <div
      className="sc-htpNat hQGAmg"
    >
      50
      %
    </div>
    <div
      className="sc-bwzfXH CuiLQ"
      style={
        Object {
          "width": "50%",
        }
      }
    >
      <div
        className="sc-bxivhb hkKilu"
        style={
          Object {
            "width": "200%",
          }
        }
      >
        50
        %
      </div>
    </div>
  </div>
</div>
`;
